api/tests/core/geography/test_models.py

```diff
@@ -23,4 +24,68 @@ def test_address_is_unique_over_street_and_insee_code(self) -> None:
                 city="Paris",
                 longitude=2.308289,
                 latitude=48.87171,
+                timezone="Europe/Paris",
+            )
+
+    def test_address_department_code_can_only_have_2_or_3_digits(self, db_session) -> None:
+        # No departementCode, should be fine as the field is nullable
+        AddressFactory(
+            street="89 Rue de la Boétie",
+            postalCode="75008",
+            inseeCode="75056",
+            city="Paris",
+            longitude=2.308289,
+            latitude=48.87171,
+            timezone="Europe/Paris",
+        )
+        # 2 digits, should be fine
+        AddressFactory(
+            street="88 Rue de la Boétie",
+            postalCode="75008",
+            inseeCode="75056",
+            city="Paris",
+            longitude=2.308288,
+            latitude=48.87170,
+            departementCode=75,
```

Comment:
C'est plutôt une `str` qui est attendue : 
```suggestion
            departementCode="75",
```